ConfigCreator/ConfigCreator/ConfigCreator.py

```diff
@@ -0,0 +1,512 @@
+        if os.path.exists(Dot_Minecraft_Path+'/versions'):
+            versions_address=os.listdir(Dot_Minecraft_Path+'/versions')
+            if versions_address:
+                for different_versions_folder in versions_address:
+                    if os.path.isdir(Dot_Minecraft_Path+'/versions/'+different_versions_folder):
+                        temp_dict={}
+                        Jar_file_found=False
+                        Native_file_found=False
+                        Json_file_found=False
+                        for temp2 in os.listdir(Dot_Minecraft_Path+'/versions/'+different_versions_folder):
+                            if os.path.isdir(Dot_Minecraft_Path+'/versions/'+different_versions_folder+'/'+temp2):
+                                for dlls in os.listdir(Dot_Minecraft_Path+'/versions/'+different_versions_folder+'/'+temp2):       #Find native dll file
+                                    if os.path.splitext(dlls)[1]=='.dll' and dlls=='jinput-dx8.dll' or dlls=='jinput-dx8_64.dll':
+                                        temp_dict['Native_files_folder']=Dot_Minecraft_Path+'/versions/'+different_versions_folder+'/'+temp2
+                                        Native_file_found=True
+                                        break
+                                    else:
+                                        continue
+                                try:           #Sometimes folder might be empty, and may cause exception.
+                                    dlls=0
+                                    del dlls
+                                except UnboundLocalError:
+                                    continue
+                            else:
+                                if os.path.splitext(temp2)[1]=='.jar':       #Find jar file
+                                    temp_dict['Jar_file']=Dot_Minecraft_Path+'/versions/'+different_versions_folder+'/'+temp2
+                                    Jar_file_found=True
+                                if os.path.splitext(temp2)[1]=='.json':      #Find json file for the jar file, if exist. It contained the address for downloading the libraries.
+                                    temp_dict['Jar_file_config']=Dot_Minecraft_Path+'/versions/'+different_versions_folder+'/'+temp2
+                                    Json_file_found=True
+
+                        # Throw out the missing version
+                        if not Jar_file_found and not Native_file_found and not Json_file_found:
+                            sys.stderr.write('{} is missing important file: Natives, Jar, Config.'.format(different_versions_folder))
+                            continue
+                        elif not Jar_file_found and not Json_file_found:
+                            sys.stderr.write('{} is missing important file: Jar, Config.'.format(different_versions_folder))
+                            continue
+                        if not Jar_file_found:
```

Comment:
Here, I change the way analyzing the version list~ The crazy forge won't be kicked now~